bin/postinstall

```diff
@@ -10,33 +10,36 @@ CLI="${APP_NAME}"
 custom_gemfile=$(${CLI} config:get CUSTOM_PLUGIN_GEMFILE || echo "")
 if [ -n "$custom_gemfile" ] && [ -f "$custom_gemfile" ]; then
 
-	# Need to override the frozen setting for the vendored gems
-	${CLI} run bundle config --local frozen 0
+        # Need to override the frozen setting for the vendored gems
+        ${CLI} run bundle config --local frozen 0
 
-	# Re-bundle the application including gems
-	${CLI} run bundle install
+        # Re-bundle the application including gems
+        ${CLI} run bundle install
 
-	# Mark assets for recompilation in the OpenProject postinstall rake step
-	${CLI} config:set MUST_REBUILD_ASSETS="true"
+        # Mark backend for recompilation in the OpenProject postinstall rake step
+        echo "Custom Gemfile present. Need to recompile rails assets. Setting RECOMPILE_RAILS_ASSETS=true"
+        ${CLI} config:set RECOMPILE_RAILS_ASSETS="true"
 fi
 
-server_prefix=$(${CLI} config:get SERVER_PATH_PREFIX || echo "/")
-if [ -n "$server_prefix" ] && [ "$server_prefix" != '/' ]; then
-	${CLI} config:set MUST_REBUILD_ASSETS="true"
+# Check for server prefix
+server_prefix=$(${CLI} config:get SERVER_PREFIX || echo "")
+if [ "$server_prefix" != "/" ]; then
+        echo "Server prefix is set. Need to recompile rails assets. Setting RECOMPILE_RAILS_ASSETS=true"
+        # Mark backend for recompilation in the OpenProject postinstall rake step
+        ${CLI} config:set RECOMPILE_RAILS_ASSETS="true"
 fi
 
-
-# If we don't have to rebuild assets, set EXECJS_RUNTIME
-must_rebuild=$(${CLI} config:get MUST_REBUILD_ASSETS || echo "")
+# Check if we need to compile angular as well
+must_rebuild=$(${CLI} config:get RECOMPILE_ANGULAR_ASSETS= || echo "")
```

Comment:
I don't see where `RECOMPILE_ANGULAR_ASSETS` is set?

Reply:
It's never set by the system. Either:

- RECOMPILE_ANGULAR_ASSETS and RECOMPILE_RAILS_ASSETS is unset, then both backend and frontend is compiled (the case when simply running `rails assets:precompile` in any environment

- The user explicitly sets RECOMPILE_ANGULAR_ASSETS because he has added a plugin that requires angular compilation. I'm not aware of any production plugin that does this, so we can assume it's not needed.